<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
        * {
            font-family: 'Verdana';
            font-size: 12px;
        }
        
    </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>800</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string>
            QLineEdit {
                color: #ecf0f1;  /* Text color */
            }
        </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Filter Torrents ...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string>
 QTabWidget::pane { 
    border: 1px solid #555;  
    background: transparent;
    height:350px;
    padding:10px;
} 
QTabBar::tab { 
    background: #3e3e3e;  
    color:  #ffd740; 
    border: 1px solid #555; 
    border: none;
    border-radius: 4px; 
} 
QTabBar::tab:selected { 
    background:  #ffd740; 
    color: #2b2b2b; 
    font-weight: bold;
} 
QTabBar::tab:hover { 
    background: #ffeb6b; 
    color: #2b2b2b; 
} 
QTabBar::tab:!selected { 
    margin-top: 2px; 
}
  </string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="label_general">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="text" stdset="0">
           <string>Upload a Torrent with Tools - Create Torrent. Download a Torrent with options in File tab.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Peers</string>
       </attribute>
       <widget class="QLabel" name="label_peers">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Loading peers...</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Content</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_tab7">
        <item>
         <widget class="QTableView" name="tableWidgetContent"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Torrent_File"/>
    <addaction name="actionAdd_Magnet_Link"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionResume"/>
    <addaction name="actionPause_All"/>
    <addaction name="actionRun_all"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCreate_Torrent_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Torrent_File">
   <property name="text">
    <string>Add Torrent File ...</string>
   </property>
  </action>
  <action name="actionAdd_Magnet_Link">
   <property name="text">
    <string>Add Magnet Link ...</string>
   </property>
  </action>
  <action name="actionPause_All">
   <property name="text">
    <string>Pause All</string>
   </property>
  </action>
  <action name="actionRun_all">
   <property name="text">
    <string>Resume All</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionCreate_Torrent">
   <property name="text">
    <string>Create Torrent</string>
   </property>
  </action>
  <action name="actionCreate_Torrent_2">
   <property name="text">
    <string>Create Torrent</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
